templates/DoorControl/index.html.ep

```diff
@@ -180,24 +150,14 @@ Welcome, <span id="name"><%= session('name') || 'stranger' %></span> from <%= $s
 
 <br />
 <div id="ofd">
```

Comment:
Do you know what "ofd" stands for?

Here a case of poor naming...  It took me a while to remember: Open Front Door (I'm pretty sure)